src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -931,4 +931,72 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     {
         return _ExpandedCommandsCache;
     }
+
+    IVector<Model::Command> _filterToSendInput(IMapView<hstring, Model::Command> nameMap,

```

Comment:
rude! this is not filtering, this is constructing a completely different set of commands. that's a different operation than filtering.